typescript/framework-extensions/vercel-ai-sdk/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@coinbase/agentkit-vercel-ai-sdk",
+  "version": "0.0.1",
```

Comment:
For a brief moment, having this version be different from that of `@coinbase/agentkit` was giving me a TypeScript error, but it has since gone away. @CarsonRoscoe can you run the vercel chatbot locally and see if you end up hitting a build error?

Reply:
I did not hit this error